include/ibamr/ibamr_enums.h

```diff
@@ -213,6 +213,38 @@ enum_to_string<LevelSetOrder>(LevelSetOrder val)
     return "UNKNOWN_LEVEL_SET_ORDER";
 } // enum_to_string
 
+/*!
+ * \brief Enumerated type for different level set time stepping schemes
+ */
+enum LevelSetTimeStepping
+{
+    GAUSS_SEIDEL_TS = 1,
```

Comment:
Can you add some documentation explaining what these do? Gauss-Seidel isn't normally thought of as a time integrator.

Reply:
Perhaps:
```suggestion
    GAUSS_SEIDEL_PSEUDO_TS = 1,
```
?

Reply:
Did that!